src/components/NavBar/SearchBar.tsx

```diff
@@ -238,15 +260,22 @@ export const SearchBar = () => {
   const searchRef = useRef<HTMLDivElement>(null)
   const { pathname } = useLocation()
   const { width: windowWidth } = useWindowSize()
+  const phase1Flag = usePhase1Flag()
 
   useOnClickOutside(searchRef, () => {
     isOpen && toggleOpen()
   })
 
-  // TODO NFT Search Results implmented here
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const collections = [] as GenieCollection[]
-  const collectionsAreLoading = false
+  const { data: collections, isLoading: collectionsAreLoading } = useQuery(
+    ['searchCollections', debouncedSearchValue],
+    () => fetchSearchCollections(debouncedSearchValue),
+    {
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      refetchOnReconnect: false,
```

Comment:
double checking these values

Reply:
this is the desired behaviour to avoid overloading api with unnecessary calls